File: Drivers/FSCC Drivers/linux/lib/c++/libs/cfscc/examples/ignore-timeout.c
#include <fscc.h> /* fscc_* */

int main(void)
{
    fscc_handle h;
    unsigned status;

    fscc_connect(0, &h);

    fscc_get_ignore_timeout(h, &status);

    fscc_enable_ignore_timeout(h);
    fscc_disable_ignore_timeout(h);

    fscc_disconnect(h);

    return 0;
}

File: Drivers/FSCC Drivers/windows/lib/fscc/net/examples/ignore-timeout.cs
using Fscc;

public class Tutorial
{
    public static int Main(string[] args)
    {
        Fscc.Port p = new Fscc.Port(0);

        var status = p.IgnoreTimeout;
        
        p.IgnoreTimeout = true;
        p.IgnoreTimeout = false;

        return 0;
    }
}